lua/hydra/lib/util.lua

```diff
@@ -187,7 +188,8 @@ function util.warn_deprecated(option, config)
       if deprecation_notice.hint then
          message = string.format("%s -- See %s", message, deprecation_notice.hint)
       end
-      vim.notify(message, vim.log.levels.WARN, {
+      -- \n turns this into a "press-enter" message (:h press-enter), it's also the only way I can see the message
+      vim.notify(message .. "\n", vim.log.levels.WARN, {
```

Comment:
I'm not quite sure why, but this message doesn't show for me on startup unless I turn it into a press enter message. I probably have some type of config messing with things. But this makes sure the message isn't missed.

Reply:
Thats odd though I actually kind of like the added friction. Deprecation warnings should be annoying.

Reply:
This is adding an extra newline to on vim.notify when using nvim-notify. I don't know that I care enough to try and "fix" it, but figured I would call it out